src/components/structures/MatrixChat.js

```diff
@@ -518,6 +519,32 @@ module.exports = React.createClass({
             case 'set_theme':
                 this._onSetTheme(payload.value);
                 break;
+            case 'unknown_device_error':
```

Comment:
Is there a reason for making these dispatched actions rather than just utility functions or something? Dispatching actions is normally to make MatrixClient or some higher level app component do something.